nuxeo-jsf/nuxeo-platform-webapp-base/src/main/java/org/nuxeo/ecm/webapp/action/WebActionsBean.java

```diff
@@ -297,6 +298,11 @@ public boolean hasCurrentTabId(String category) {
 
     @Override
     public void setCurrentTabId(String category, String tabId, String... subTabIds) {
+        if (category != null && category.equals(MAIN_TABS_CATEGORY) && tabId != null
+                && currentTabActions.getCurrentTabIds() != null
+                && !currentTabActions.getCurrentTabIds().startsWith(category + ":" + tabId)) {
```

Comment:
Why not using `!tabId.equals(getCurrentTabId(MAIN_TABS_CATEGORY))`?

Reply:
Thanks for the suggestion. I changed my code and tested that it works fine like that.